tests/tensor_search/integ_tests/test_search_unstructured.py

```diff
@@ -1375,3 +1375,20 @@ def test_lexical_query_can_not_be_none(self):
                 with self.assertRaises(InvalidArgError):
                     res = tensor_search.search(text=None, config=self.config, index_name=self.default_text_index,
                                                search_method=SearchMethod.LEXICAL)
+                    
+    def test_empty_lexical_query(self):
+        tensor_search.add_documents(config=self.config,
+                                            add_docs_params=AddDocsParams(
+                                                index_name=self.default_text_index,
+                                                docs=[
```

Comment:
Can we make a test that shows that multiple documents can be returned like this? Just to show the query will return all documents, not just 1.

Reply:
done!